### DISCUSSION TITLE
soluzione per dimmerare luci led 1-10v

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2883] [HUB: Home Assistant]
Ciao e anticipatamente grazie per ogni suggerimento che possiate darmi...

avrei necessità di dimmerare dei pannelli led (dimmerabili) per controllarli con HA e per questo avevo pensato al Fibaro Dimmer 2 già più volte usato con quale mi trovo benissimo ... solo che mi sono imbattuto in un problema scoperto successivamente.

i pannelli erano stati acquistati da mio padre un paio di anni fa ed ho scoperto che si sono dimmerabili ma necessitano di un dimmer 1-10v quindi il Fibaro (sembrerebbe non più idoneo)
avete qualche suggerimento per ottenere risultato positivo alla mia esigenza? il top sarebbe poter mantenere anche il fibaro.

grazie grazie grazie

**Comment by nrizzano** [comment]
Ciao, io ho utilizzato uno [Shelly Dimmer 0-10v](https://indomus.it/link/shelly-plus-0-10v-dimmer/) e si integra perfettamente all'interno di Home Assistant.

**Reply by effe-7** [reply]
Ciao
Ma lo shelly che mi dici vale anche per 1-10v o solo 0-10v
Grazie
